notebooks/1.Getting_Data_Ready.ipynb

```diff
@@ -71,8 +103,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "forecast = boto3.client(service_name='forecast') \n",
-    "forecastquery = boto3.client(service_name='forecastquery')"
+    "session = boto3.Session(region_name=region) \n",
+    "forecast = session.client(service_name='forecast') \n",
+    "forecastquery = session.client(service_name='forecastquery')"
    ]
   },
   {
```

Comment:
Line 144: Spelling of *different* instead of *differennt*

Reply:
Addressed by the latest commit.